app/main.py

```diff
@@ -1,6 +1,15 @@
-# write your imports here
+from app.cinema.bar import CinemaBar
+from app.cinema.hall import CinemaHall
+from app.people.cinema_staff import Cleaner
+from app.people.customer import Customer
 
 
 def cinema_visit(customers: list, hall_number: int, cleaner: str, movie: str):
-    # write you code here
-    pass
+    hall_number = CinemaHall(hall_number)
+    cleaner = Cleaner(cleaner)
+    custom_list = []
```

Comment:
```suggestion
    customer_list = []
```
`custom` - is a different word, so it's better not to shorten